integration_tests/tests/features/Chat.feature

```diff
@@ -0,0 +1,43 @@
+# Copyright 2023 The Tari Project
+# SPDX-License-Identifier: BSD-3-Clause
+
+Feature: Chat messaging
+
+  Scenario: A message is propagated between nodes
+    Given I have a seed node SEED_A
+    When I have a chat client CHAT_A connected to seed node SEED_A
+    When I have a chat client CHAT_B connected to seed node SEED_A
+    When I use CHAT_A to send a message 'Hey there' to CHAT_B
+    Then CHAT_B will have 1 message with CHAT_A
+
+  Scenario: A message is sent directly between nodes
+    Given I have a seed node SEED_A
+    When I have a chat client CHAT_A connected to seed node SEED_A
+    When I have a chat client CHAT_B connected to seed node SEED_A
```

Comment:
Is it still necessary to have the 3rd party base node here?

Reply:
Without the seed node the two chat clients don't seem to find each other after being added as contacts. We would probably need to introduce a peer dialing step for new contacts if the client doesn't otherwise have connectivity.